src/p5.dance.js

```diff
@@ -92,7 +92,7 @@ module.exports = class DanceParty {
     // Sort after spriteConfig function has executed to ensure that
     // rest moves are at the beginning and shortBurst moves are all at the end
     this.world.MOVE_NAMES = this.world.MOVE_NAMES.sort((move1, move2) => (
-      move1.rest * -2 * move2.rest * 2 + move2.shortBurst * -1 + move1.shortBurst * 1
+      !!move1.rest * -2 * !!move2.rest * 2 + !!move2.shortBurst * -1 + !!move1.shortBurst * 1
```

Comment:
Shouldn't this be
```suggestion
      !!move1.rest * -2 + !!move2.rest * 2 + !!move2.shortBurst * -1 + !!move1.shortBurst * 1
```
As-is you've got:
![image](https://user-images.githubusercontent.com/1615761/48213714-47120280-e333-11e8-83f8-344ba8233806.png)
But it should be:
![image](https://user-images.githubusercontent.com/1615761/48213735-55601e80-e333-11e8-9507-4ab46b0ba925.png)

Reply:
thanks for catching that. hence, the need to have more test cases.